chat.go

```diff
@@ -218,6 +218,18 @@ type ChatCompletionRequest struct {
 	ToolChoice any `json:"tool_choice,omitempty"`
 	// Options for streaming response. Only set this when you set stream: true.
 	StreamOptions *StreamOptions `json:"stream_options,omitempty"`
+	// Disable the default behavior of parallel tool calls.
+	ParallelToolCalls *bool `json:"parallel_tool_calls,omitempty"`
```

Comment:
I guess we can make this an `any`, comment that the boolean is expected and don't bother with `ParallelOptionTrue`

Reply:
Yes! makes much more sense.

have modified it to be any